precise/scripts/convert.py

```diff
@@ -46,42 +45,33 @@ def convert(self, model_path: str, out_file: str):
             out_file: location to write protobuf
         """
         print('Converting', model_path, 'to', out_file, '...')
-
-        import tensorflow as tf
+        
+        import tensorflow as tf # Using tensorflow v2.0
+        from tensorflow import keras as K
         from precise.model import load_precise_model
-        from keras import backend as K
-
+        from precise.functions import weighted_log_loss
+        
         out_dir, filename = split(out_file)
         out_dir = out_dir or '.'
         os.makedirs(out_dir, exist_ok=True)
-
-        K.set_learning_phase(0)
-        model = load_precise_model(model_path)
-
-        out_name = 'net_output'
-        tf.identity(model.output, name=out_name)
-        print('Output node name:', out_name)
-        print('Output folder:', out_dir)
-
-        sess = K.get_session()
-
-        # Write the graph in human readable
-        tf.train.write_graph(sess.graph.as_graph_def(), out_dir, filename + 'txt', as_text=True)
-        print('Saved readable graph to:', filename + 'txt')
-
-        # Write the graph in binary .pb file
-        from tensorflow.python.framework import graph_util
-        from tensorflow.python.framework import graph_io
-
-        cgraph = graph_util.convert_variables_to_constants(sess, sess.graph.as_graph_def(), [out_name])
-        graph_io.write_graph(cgraph, out_dir, filename, as_text=False)
-
-        if isfile(model_path + '.params'):
-            copyfile(model_path + '.params', out_file + '.params')
-
-        print('Saved graph to:', filename)
-
-        del sess
+        
+        # Load custom loss function with model
+        model = K.models.load_model(model_path, custom_objects={'weighted_log_loss': weighted_log_loss})
+        
+        model.summary()
+        
+        # Support for freezing models to .pb has been removed in TF 2.0.
+        
+        # Converting instead to TFLite model
+        print('Starting TFLite conversion.')
+        converter = tf.lite.TFLiteConverter.from_keras_model(model)
+        converter.target_ops = [tf.lite.OpsSet.TFLITE_BUILTINS,tf.lite.OpsSet.SELECT_TF_OPS]
+        #  converter.experimental_new_converter=True # TF2.2 uses experimental converter by default.
```

Comment:
Here

Reply:
Added this for documentation purposes, we can remove this without a problem.

Reply:
Addressed this is latest commit 'General cleanup for merge'.